apollo-router/src/plugins/file_uploads/mod.rs

```diff
@@ -254,7 +254,7 @@ fn replace_value_at_path<'a>(
 
 // Removes value at path.
 fn remove_value_at_path<'a>(variables: &'a mut json_ext::Object, path: &'a [String]) {
-    let _ = get_value_at_path(variables, path).take();
+    let _ = get_value_at_path(variables, path).map(|v| *v = serde_json_bytes::Value::Null);
```

Comment:
I think this would better communicate intent
```suggestion
    if let Some(v) = get_value_at_path(variables, path) {
        *v = serde_json_bytes::Value::Null;
    }
```

Reply:
Love it 😍 
Both you and @goto-bus-stop normalized writing procedural code in Rust in your comments.
I was pushing myself to use a functional style in Rust, which I'm not fond of.
Thanks a lot 👍